plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergFileOperations.java

```diff
@@ -905,6 +916,43 @@ public void testShowTables()
         assertFileSystemAccesses("SHOW TABLES", ImmutableMultiset.of());
     }
 
+    @Test
```

Comment:
This test class is for verifying file operation counts. Please move to `TestIcebergV2` which has `loadTable` method or somewhere.